crates/astria-gossipnet/tests/network.rs

```diff
@@ -16,6 +16,20 @@ use tokio::{
 
 const TEST_TOPIC: &str = "test";
 
+#[tokio::test]
+async fn keypair_from_file() {
+    let key_string = "241ccaff3cc8681385b2cf92a82b49ab2e9c6410f7b6191322867c4299de42d8";
+    let path = std::env::temp_dir().join("test.key");
```

Comment:
`std::env::temp_dir()` returns `/tmp`. In tests it's better to use [`tempfile::tempfile](`https://docs.rs/tempfile/latest/tempfile/fn.tempfile.html) which ensures that the file gets deleted by the OS after the file handle is closed (make sure to not drop the handle until after the test is done).